crates/ln-dlc-node/src/node/ln_channel.rs

```diff
@@ -77,6 +79,18 @@ impl<P> Node<P> {
                         )
                         .map_err(|e| anyhow!("Could not force close channel {e:?}"))
                 } else {
+                    if let Some(SubChannel { state, .. }) = self.list_dlc_channels()?.first() {
+                        if *state != SubChannelState::OffChainClosed {
+                            // this is a safeguard to not close a ln channel with
+                            // an open dlc, as we would loose the funds otherwise.
+
+                            // todo: Ideally we would start to collaborative close the dlc
```

Comment:
can we please stick to uppercase letters? This variant is picked up by most IDEs automatically, maximising chances of going through this TODO. 

```suggestion
                            //TODO: Ideally we would start to collaborative close the dlc
```